packages/gatsby-plugin-gatsby-cloud/src/components/Style.js

```diff
@@ -27,6 +27,10 @@ const Style = () => (
             0px 4px 8px rgba(71, 63, 79, 0.16);
         }
 
+        svg {
+          display: inline;
+        }
```

Comment:
This is dangerous as you overwrite it for all sites. Please use a className

Reply:
This is in a shadow root. So it only applies to the preview indicator.